README.md

```diff
@@ -200,11 +200,30 @@ ResponseCache::forget('/some-uri');
 // Forget several URIs
 ResponseCache::forget(['/some-uri', '/other-uri']);
 
+// Using Tags (the cache driver you configured must supports tags)
+ResponseCache::forget(['/some-uri', '/other-uri'], ['tag1', 'tag2']);
```

Comment:
It's not clear what this does.

Possibilities: 
- Will it only forget the uri's if they have one of the given tags?
- Will it only forget the uri's if they have both of the given tags?
- Will it only forget the uri's and everything else with one of the given tags?
- Will it only forget the uri's and everything else with all of the given tags?

This could be made less ambitious by avoiding using a second argument to `forget`.

Maybe you could introduce dedicated methods: `ResponseCache::forgetWithAnyTag`,  `ResponseCache::forgetWithAllTags`.

Reply:
I not added the tag parameter, it was alredy present in the ResponseCache::forget() functiony.
I simply added the line in the readme and replicated the function in the cache cleaner.

Reply:
Actually you have right, having tags as second argument make no sense:
https://laravel.com/docs/8.x/cache#removing-tagged-cache-items

I think that forget function can be modified in this way:
 ```
 public function forget(string | array $uris): self
  {}
```
It seems that do not break any test.

Reply:
@freekmurze 
I was wrong, to remove tagged object from cache is mandatory to specify tags:

So better to leave the ResponseCache::forget() function with the old signature, to mantain backward compatibility.
```
    public function forget($uris, array $tags = []): self
    {
        $uris = is_array($uris) ? $uris : func_get_args();
        //....
     }

```